climpred/bootstrap.py

```diff
@@ -24,24 +24,24 @@
 )
 
 
-def _resample(hind, shuffle_dim, to_be_shuffled):
-    """Resample with replacement in dimension `shuffle_dim` from values of
-    `to_be_shuffled`
+def _resample(hind, resample_dim, to_be_resampled):
+    """Resample with replacement in dimension `resample_dim` from values of
+    `to_be_resampled`
 
     Args:
         hind (xr.object): input xr.object to be shuffled.
```

Comment:
If you're removing `shuffle` language above in favor of `resample`, maybe change all instances of "shuffle" here to "resample"?

Reply:
resample is more correct. I will change.